src/client/app/app-routing.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  imports: [
+    RouterModule.forRoot([
+      /* define app non-feature routes here */
```

Comment:
I am fairly new to angular and I figured it is a good practice to switch the routing to the new style guide. [In some places](https://angular.io/docs/ts/latest/guide/router.html#!#the-heroes-app-code) in the doc the app-routing.module.ts is more or less the same as the one here -> empty. In another place it is used for [lazy loading modules](https://angular.io/docs/ts/latest/guide/router.html#!#lazy-loading-route-configuration).